gravitee-apim-repository/gravitee-apim-repository-mongodb/src/main/java/io/gravitee/repository/mongodb/management/internal/model/UserMongo.java

```diff
@@ -26,6 +29,10 @@
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
+@Setter
+@Getter
+@ToString
```

Comment:
For ApiKeyMongo.java you included only a few fields to match the current toString() method. Here you went with the default @ToString. I'm ok with that. I'm just wondering if that was on purpose or you just missed that :)